scripts/Muon/GUI/Common/plot_widget/plotting_canvas/plotting_canvas_view.py

```diff
@@ -207,24 +207,62 @@ def set_axis_ylimits(self, axis_number, ylims):
     def set_axes_limits(self, xlim, ylim):
         plt.setp(self.fig.axes, xlim=xlim, ylim=ylim)
 
+    def is_error_plotted(self):
+        #Function is defined in the presenter
```

Comment:
I would have expected this to havet returned the state of the error checkbox. Why is it defined in the presenter?

Reply:
The plotting canvas view does not have access to quick edit widget but the presenter does.

Reply:
could you add that to the comment please?